src/app/pages/home-page/home-page.component.scss

```diff
@@ -0,0 +1,84 @@
+.wrapper-home-page {
+  height: calc(100vh - 110px);
+}
+
+.wrapper-content {
+  height: 100%;
+  background: rgba(37, 51, 73, 0.85);
+}
+
+.wrapper-drop-box,
+.wrapper-description {
+  width: 100%;
+  padding: 6px 0 0 0;
+  display: flex;
+  justify-content: center;
+}
+
+.wrapper-drop-box {
+  height: 5%;
+}
+
+.wrapper-description {
+  height: 15%;
+}
+
+.drop-box,
+.description {
+  height: 100%;
+  width: calc(100% - 234px);
```

Comment:
I think things like width should also be moved into variable.

Reply:
done